src/d3a/setup/two_sided_market/infinite_bus.py

```diff
@@ -18,38 +18,25 @@
 from d3a.models.appliance.switchable import SwitchableAppliance
 from d3a.models.area import Area
 from d3a.models.strategy.load_hours import LoadHoursStrategy
-from d3a.models.strategy.infinite_bus import InfiniteBusStrategy
+# from d3a.models.strategy.infinite_bus import InfiniteBusStrategy
+from d3a.models.strategy.market_maker_strategy import MarketMakerStrategy
 from d3a.models.appliance.simple import SimpleAppliance
-from d3a.models.strategy.pv import PVStrategy
-from d3a.models.appliance.pv import PVAppliance
+# from d3a.models.strategy.pv import PVStrategy
+# from d3a.models.appliance.pv import PVAppliance
 from d3a_interface.constants_limits import ConstSettings
 
 
 def get_setup(config):
-    ConstSettings.IAASettings.MARKET_TYPE = 2
+    ConstSettings.IAASettings.MARKET_TYPE = 1
     area = Area(
         'Grid',
         [
-            Area(
-                'House 1',
-                [
-                    Area('H1 General Load', strategy=LoadHoursStrategy(avg_power_W=200,
-                                                                       hrs_of_day=list(
-                                                                           range(0, 23)),
-                                                                       final_buying_rate=27,
-                                                                       initial_buying_rate=27),
-                         appliance=SwitchableAppliance()),
-                    Area('H1 PV', strategy=PVStrategy(panel_count=4,
-                                                      final_selling_rate=15,
-                                                      initial_selling_rate=24.1),
-                         appliance=PVAppliance()),
-                ], transfer_fee_const=1
-            ),
-            Area('Infinite Bus', strategy=InfiniteBusStrategy(energy_buy_rate=24,
-                                                              energy_sell_rate=24),
-                 appliance=SimpleAppliance()
-                 ),
-
+            Area('Market Maker', strategy=MarketMakerStrategy(energy_rate=30),
```

Comment:
So, this is not testing the infinite bus any more, so either revert the changes or rename the file. Also, please remove lines that are commented. Thanks

Reply:
Thanks for pointing it out, have reverted to its original state.